applications/NXmx.nxdl.xml

```diff
@@ -816,6 +822,27 @@
 						</doc>
 					</field>
 
+					<field name="flux_integrated" type="NX_FLOAT" units="NX_FLUX" minOccurs="0">
```

Comment:
unit should be per area (NX_PER_AREA)

Reply:
Merge https://github.com/nexusformat/definitions/pull/1082 to fix.